client/flutter/lib/api/question_data.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:WHOFlutter/api/content_loading.dart';
+import 'package:flutter/cupertino.dart';
+
+class QuestionData {
+  static Future<List<QuestionItem>> yourQuestionsAnswered(
+      BuildContext context) async {
+    var bundle =
+        await ContentLoading().load(context, "your_questions_answered");
+    try {
+      return bundle.contents
+          .map((item) =>
+              QuestionItem(title: item['title_html'], body: item['body_html']))
+          .toList();
```

Comment:
Blindly trust network contents is dangerous. A few edge cases to consider:

- One of the `item` is not a map
- `item` doesn't have a `title_html` or `body_html` or both (!)

I think we should do some basic checks to avoid one bad item ruining the entire content bundle.

```suggestion
      return bundle.contents
          .where((item) =>
              item is Map &&
              item.containsKey('title_html') &&
              item.containsKey('body_html'))
          .map((map) => QuestionItem(
                title: map['title_html'].toString(),
                body: map['body_html'].toString(),
              ))
          .toList();
```